eox_tenant/filters/README.rst

```diff
@@ -13,8 +13,14 @@ Filters steps list:
 -------------------
 
 * `FilterUserCourseEnrollmentsByTenant`_: Filters the course enrollments of a user from the tenant site where the request is made.
-
-.. _FilterUserCourseEnrollmentsByTenant: ./pipeline.py#L9
+* `FilterRenderCertificatesByOrg`_: Stop certificate generation process raising a exception if course org is different to tenant orgs.
+* `OrgAwareLMSURLStudio`_: Filter specific org aware link form Studio to the LMS.
+* `OrgAwareCourseAboutPageURL`_: Filter specific org aware course about page URL from Studio to the LMS.
```

Comment:
I think it would be easier to follow if we explicitly explained what each filter does without using org-aware or filter terminology. Does this sound right to you? 
```suggestion
* `OrgAwareLMSURLStudio`_: Generates a new LMS URL for asset URL generation based on the course organization settings.
* `OrgAwareCourseAboutPageURL`_: Generates a new course about URL based on the course organization settings.
```